examples/Org.OData.Aggregation.V1.SalesModel-sample.xml

```diff
@@ -109,6 +109,59 @@
           <NavigationPropertyBinding Path="Time" Target="Time" />
         </EntitySet>
       </EntityContainer>
+      <Annotations Target="SalesModel.Container/Sales">
+        <Annotation Term="ApplySupported">
+          <Record>
+            <PropertyValue Property="Rollup" EnumMember="Aggregation.RollupType/MultipleHierarchies" />
+            <PropertyValue Property="GroupableProperties">
+              <Collection>
+                <PropertyPath>SalesOrganization/ID</PropertyPath>
+                <PropertyPath>SalesOrganization/Name</PropertyPath>
+                <PropertyPath>Product/ID</PropertyPath>
+                <PropertyPath>Product/Name</PropertyPath>
+                <PropertyPath>Product/Color</PropertyPath>
+                <PropertyPath>Product/Category/ID</PropertyPath>
+                <PropertyPath>Product/Category/Name</PropertyPath>
+                <PropertyPath>Customer/ID</PropertyPath>
+                <PropertyPath>Customer/Name</PropertyPath>
+                <PropertyPath>Customer/Country</PropertyPath>
+                <PropertyPath>Time/Date</PropertyPath>
```

Comment:
TC 2022-04-14: Examples contain `groupby((Time))`, which is allowed if and only if `groupby((Time/Date))` (the key proeprty) is allowed. `GroupableProperties` only contain the properties, not the navigation properties. Call this out in the vocabulary. Alternatively, allow
```suggestion
                <NavigationPropertyPath>Time</NavigationPropertyPath>
                <PropertyPath>Time/Date</PropertyPath>
```
(changing the vocabulary to 4.01 and `Edm.AnyPropertyPath`).